docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     volumes:
       - ./conf/mariadb-conf.d:/etc/mysql/conf.d
       - /var/lib/mysql
+        #      - ./db/var/lib/mysql for data persistence
```

Comment:
That line isn't needed, as it's already covered by 

```yaml
    - /var/lib/MySQL
```

Reply:
Better this way ?

Reply:
The less lines the better, and the less confusing